security/v1/types.go

```diff
@@ -351,3 +351,34 @@ type ServiceAccountPodSecurityPolicyReviewStatus struct {
 	// name contains the allowed and the denied ServiceAccount name
 	Name string `json:"name" protobuf:"bytes,2,opt,name=name"`
 }
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// RangeAllocation is used so we can easily expose a RangeAllocation typed for security group
+type RangeAllocation struct {
+	metav1.TypeMeta `json:",inline"`
+	// Standard object's metadata.
+	// More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#metadata
+	// +optional
+	metav1.ObjectMeta `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
+
+	// Range is string that identifies the range represented by 'data'.
+	Range string `json:"range" protobuf:"bytes,2,opt,name=range"`
```

Comment:
it's not clear what the difference is intended to be between this and name... example or clarify in doc here?